do/droplets.go

```diff
@@ -91,7 +91,9 @@ func (i *instances) ExternalID(nodeName types.NodeName) (string, error) {
 
 // InstanceID returns the cloud provider ID of the node with the specified NodeName.
 func (i *instances) InstanceID(nodeName types.NodeName) (string, error) {
+
```

Comment:
Remove this empty line after the function name, to conform to the rest of the project style